sycl/test/reduction/reduction_nd_s0_dw.cpp

```diff
@@ -21,6 +21,10 @@ class SomeClass;
 
 template <typename T, int Dim, class BinaryOperation>
 void test(T Identity, size_t WGSize, size_t NWItems) {
+  queue Q;
+  if (!isSupportedType<T>(Q.get_device()))
```

Comment:
Not for this MR though @bader  brought up restructuring the test system in the future and this is an example of us probably wanting feature detection to not even need to run testst that require specific features/extensions.